storage/kubernetes/client.go

```diff
@@ -58,11 +58,24 @@ func (c *client) idToName(s string) string {
 	return idToName(s, c.hash)
 }
 
+// idToNameComposite maps two arbitrary IDs, to a single Kubernetes object name.
+// This is used when more than one field is used to uniquely identify the object.
+func (c *client) idToNameComposite(s1 string, s2 string) string {
```

Comment:
Made it generic incase other objects wanted to use it later on. But will make it userID and connID for now since only OfflineSession makes use of it.